pkg/virt-controller/watch/vm_test.go

```diff
@@ -3646,7 +4041,7 @@ var _ = Describe("VirtualMachine", func() {
 				// We need to clear the domainSpec here to ensure the instancetype doesn't conflict
 				vm.Spec.Template.Spec.Domain = v1.DomainSpec{}
 
-				fakeInstancetypeClients = fakeclientset.NewSimpleClientset().InstancetypeV1beta1()
+				fakeInstancetypeClients = fake.NewSimpleClientset().InstancetypeV1beta1()
```

Comment:
Couldn't use `fakeInstancetypeClients = virtFakeClient.InstancetypeV1beta1()`?

Reply:
I'll leave it for future.